### DISCUSSION TITLE
the mascot name?

### DISCUSSION THREAD

**Discussion by atesin** [nesbox/TIC-80#2389] [Q&A]
hi.. what is the name of tic90 mascot? ..  that little blue pc?

would be nice to see this character more present in games, but for some reason is failing, maybe lack of personality, "cutiness", or who knows  (actually i think this is the first time is mentioned in whole internet)

**Comment by The-Beesh-Spweesh** [comment]
I think it's called Ticsy.

**Comment by atesin** [comment]
"Ticsy" ... sure?

would be nice to invent a little story about him/her :) ... who invented this little boxy blueish pc? ... on what circumstances? .. i am curious

besides being the "brand image", what can @nesbox say abour our friend?  ... along with some drawings/3d models to use him/her in our games if possible

**Comment by ElidavididiOnGithub** [comment]
So, Pico-8 Has Jelpi, And Tic-80 Has Ticsy, Now The Real Crime Is Pixelvision8's Mascot
